web/packages/design/src/StepSlider/StepSlider.tsx

```diff
@@ -323,12 +323,18 @@ type Props<T> = {
 };
 
 export type StepComponentProps = {
-  // refCallback is a func that is called after component mounts.
-  // Required to calculate dimensions of the component for height animations.
+  /**
+   * refCallback is a func that is called after component mounts.
+   * Required to calculate dimensions of the component for height animations.
+   */
   refCallback(node: HTMLElement): void;
-  // next goes to the next step in the flow.
+  /**
+   * next goes to the next step in the flow.
```

Comment:
I don't really care about it, but it's the standard in Go. https://go.dev/wiki/CodeReviewComments#comment-sentences

The language server I use for Go even updates the name of a function in the comment after I rename a function.

Since JSDoc is pretty flexible in that regard and the Go community isn't, I'd rather accept that some JSDocs will repeat the name of the thing they describe.

Reply:
> is pretty flexible in that regard and the Go community isn't

typical